### PR TITLE
Avoid reconfiguring VPN when game is installed

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/488551667048375/1203238561022240/f

### Description
Avoid re-starting AppTP when games are installed, see [asana task](https://app.asana.com/0/488551667048375/1203238561022240/f) for more details

### Steps to test this PR

_Test install games_
- [ ] install from this branch
- [ ] enable AppTP
- [ ] install a game, eg. candy crash
- [ ] verify VPN is not restarted

### CODE REVIEW

**Comment by aitorvs** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #2453** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/2453" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/2453?utm_source=stack-comment).

File: app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/bugreport/VpnAppTrackerListInfoCollector.kt [hunk 1]
```diff
@@ -49,8 +52,11 @@ class VpnAppTrackerListInfoCollector @Inject constructor(
     }
 
     private fun isUnprotectedByDefault(appPackageId: String): Boolean {
-        return appCategoryDetector.getAppCategory(appPackageId) is AppCategory.Game ||
-            appTrackerRepository.getAppExclusionList().any { it.packageId == appPackageId }
+        return isGame(appPackageId) || appTrackerRepository.getAppExclusionList().any { it.packageId == appPackageId }
+    }
+
+    private fun isGame(packageName: String): Boolean {
+        return appCategoryDetector.getAppCategory(packageName) is AppCategory.Game && !appTpFeatureConfig.isEnabled(AppTpSetting.ProtectGames)
```
Comment (malmstein) [inline app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/bugreport/VpnAppTrackerListInfoCollector.kt:59]: nit: can we have create an `appTpFeatureConfig.isDisabled` to make it clearer?

**Review by malmstein — APPROVED** [review]
:shipit:

File: app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/bugreport/VpnAppTrackerListInfoCollector.kt (hunk 1 above)
Comment (aitorvs) [reply app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/bugreport/VpnAppTrackerListInfoCollector.kt:59]: Will do in different PR so refactor all usages if `isEnabled`

**Pull request merged**